react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Is there any way to note that this applies to version 3 of `react-hot-loader`?

Version 1 has a completely different API (and a lot of disadvantages, such as only supporting createClass components).